.github/workflows/devbuild.yml

```diff
@@ -423,11 +445,26 @@ jobs:
           Get-Command clang-tidy
           Get-Command flake8
 
-      - name: cmake ALL_BUILD
+      - name: cmake ALL_BUILD USE_PYTEST_HELPER_BINDING=OFF DEBUG_BUILD_OPTIMIZATION_ONE=ON
```

Comment:
Here's an instance where `USE_PYTEST_HELPER_BINDING` is set to `OFF`. While there are additional scenarios, I'm highlighting one for reference.

`DEBUG_BUILD_OPTIMIZATION_ONE` is `ON` for both the debug build and the release build, but it only works for the debug build. I don't think we need to have a special condition branch to separate the two builds.